Update ls.md by adding ls -F (#4071)
tldr-pages/tldr: pages/common/ls.md

`ls -a`

> - List all files with trailing `/` indicates that this is a directory:

Reviewer: ```suggestion
- List all files, with trailing `/` added to directory names:
```
